ompi/tools/mpirun/Makefile.am

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019      Intel, Inc.  All rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+
+
+bin_SCRIPTS = mpirun
```

Comment:
Is this meant to be a bash script or a link to `prun`, like it is in PRRTE?

Reply:
Now I understand the confusion - you are looking at a totally different place. I will remove that directory as it is no longer being used. PRRTE installs after it and simply blows that script away, replacing it with ```prun```

Reply:
I thought we wanted OMPI to own the `mpirun` tool - since PRRTE is not MPI specific. From that perspective, the soft link should probably be in this location in OMPI, and removed from PRRTE.

Reply:
If you have time, then sure - that is fine with me. Please update the respective Makefile.am's (you'll need to do PRRTE and OMPI).

Reply:
Yep. I'll have a patch up in a bit. I'll post back when they are ready.

Reply:
Two PRs ready for review:
 * PRRTE: https://github.com/openpmix/prrte/pull/287
 * OMPI (to this branch): https://github.com/rhc54/ompi/pull/9